media-gfx/ahoviewer/ahoviewer-1.6.2.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+GNOME2_EAUTORECONF="yes"
+
+inherit gnome2 xdg-utils
+
+DESCRIPTION="A GTK2 image viewer, manga reader, and booru browser"
+HOMEPAGE="https://github.com/ahodesuka/ahoviewer"
+SRC_URI="https://github.com/ahodesuka/ahoviewer/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl_ssl_gnutls +curl_ssl_openssl +gstreamer libsecret +rar +ssl +zip"
+
+DEPEND="
+	dev-cpp/glibmm:2
+	dev-cpp/gtkmm:2.4
+	dev-libs/libconfig[cxx]
+	dev-libs/libxml2
+	dev-libs/libsigc++:2
+	net-misc/curl:=
+	gstreamer? (
+		media-libs/gst-plugins-bad:1.0
+		media-libs/gstreamer:1.0
+	)
+	libsecret? ( app-crypt/libsecret )
+	rar? ( app-arch/unrar )
+	ssl? (
+		net-misc/curl[curl_ssl_gnutls(-)=,curl_ssl_openssl(-)=]
```

Comment:
I'm confused. Does it actually link to GnuTLS/OpenSSL, or what does it do?

Reply:
It doesn't seem to do so, 
```
joonas@starlight ~ $ readelf -d /usr/bin/ahoviewer | grep NEEDED | grep -i ssl
joonas@starlight ~ $ 
joonas@starlight ~ $ ldd /usr/bin/ahoviewer | grep -i ssl
	libssl.so.1.0.0 => /usr/lib64/libssl.so.1.0.0 (0x00007f72e4e44000)

```